keps/sig-storage/3751-volume-attributes-class/README.md

```diff
@@ -439,31 +439,28 @@ Operation metrics from [csiOperationsLatencyMetric](https://github.com/kubernete
 
 While a VolumeAttributesClass is referenced by any PVC, we will prevent the object from being deleted by adding a finalizer([reference](https://github.com/kubernetes/kubernetes/blob/master/plugin/pkg/admission/storage/storageobjectinuseprotection/admission.go)). 
```

Comment:
I would also mention in the beginning of this section that the finalizer is going to be best effort only and prone to races.